### DISCUSSION TITLE
22uF Tantalum has no polarity, and NO negative. /How can I get the data out of the NES file?

### DISCUSSION THREAD

**Discussion by SAVE-THE-HERO** [sanni/cartreader#238] [Need help]
Hi
Thank you for so cool project.
I love it!!

Q1.
I bought a 22uF Tantalum 0805 from the Needed-Parts section of the wiki at the e-bay link.
The one delivered to me is indeed 22uF 0805, but it has no polarity, and NO negative and NO Tantalum.
Is it safe to use this?
<img width="587" alt="2021-08-31 15 14 26" src="https://user-images.githubusercontent.com/89830526/131456674-43d17dd0-3be5-4101-8bdb-346b1b3cf0b1.png">

Q2.
How can I get the data out of the NES?
I have watched sanni's youtube.
In the video it says "NES FILE OUT PUT!
On my display the PRG, CHR and RAM are DAMPED and the BIN is OUTPUT instead of the NES.
I have tried with 2 different FC cartridges with the same result.

How can I get the NES and save data out?
I upload my video.
https://u.pcloud.link/publink/show?code=XZJgXdXZOzsslv1Avw0MOqxhYVhhiSYqHpLV

Thank you!!!

**Comment by sanni** [comment]
Q1: What does the markings on the other side say? 

Q2: If just the bin is generated then the game was not found in the NES database and you need to manually add the iNES header to turn the bin into a nes file.

You can use a program on your PC:

    https://github.com/dsedivec/inestool
    https://greg-kennedy.com/wordpress/2012/05/30/ines-header-fixer/

Or use a hex editor to prepend the 16 byte header from here:

    http://nes.dnsabr.com/

**Reply by SAVE-THE-HERO** [reply]
Hi sanni.
Thank you for reply!!
Q1.
oh,,,sorry and thank you sanni, I can see polarity line.
In this picture, the line to the left of the capacitor is a positive.
So This is Tantalum!!
(Please let me know if I'm wrong)
![IMG_3249](https://user-images.githubusercontent.com/89830526/131542727-9c89a2bf-5eb3-40ea-b945-a063fef1820e.jpg)

Q.2
Thank you
I wii try it....

By the way I'm making a special case for cartreader.
It's a prototype and I can't show it to you at the moment, but hopefully you'll see it soon!

Many thanks for your support.
Thank you!!
